app/error_handlers.py

```diff
@@ -30,4 +30,20 @@ def method_not_allowed(e):
 
 @app.errorhandler(500)
 def internal_error(e):
-    return make_response(jsonify(status=500, error='Internal Server Error', message='Well, this is embarrassing...'), status.HTTP_500_INTERNAL_SERVER_ERROR)
+	return make_response(jsonify(status=500, error='Internal Server Error', message='Well, this is embarrassing...'), status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+class InvalidPaymentID(Exception):
+    status_code = 404
```

Comment:
I think i missed to make a comment that test cases for this exception is in progress and will mostly be part of next sprint - taking forever to figure out some mocking errors